indicators/tokenup-crypto-scam.yml

```diff
@@ -0,0 +1,32 @@
+title: Solana cryptocurrency wallet drainer - tokenup
+
+description: |
+    Detects a Solana cryptocurrency wallet drainer that fakes the number
+    of minted NFTs to initiate Fear of Missing Out (FOMO) against the victim.
+
+author: iamdeadlyz
+    
+references:
+    - https://urlscan.io/search/#filename%3A%22tokenup.js%22
+    - https://urlscan.io/result/e72a0ab1-adfb-4e34-871d-6c9903c1e7e0/
+    - https://urlscan.io/result/c1c9e6f2-c66b-4545-acf3-f5ceec21f6b0/
+     
+detection:
+
+    tokenupJsFile:
+      requests|contains: 'tokenup.js'
+
+    settingsJsFile:
+      requests|contains: 'settings.js'
+
+    mintNumber:
+      html|contains: 'tokennumber'
+
+    blockchainIdentifier:
+      html|contains: 'solana'
+      
+    condition: (tokenupJsFile and settingsJsFile) or (tokenupJsFile and mintNumber) or (tokenupJsFile and blockchainIdentifier)
```

Comment:
I think it is possible to shorten this condition to the following:
`tokenupJsFile and (settingsJsFile or mintNumber or blockchainIdentifier)`

Reply:
Oh, good idea!